# list of port types
# port_type_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 input_INVALID  = 0 #  Invalid input type
	uint8 input_UART     = 1 #  UART COM port
	uint8 input_SDLC     = 2 #  SDLC Input port
	uint8 input_ETHERNET = 3 #  ETHERNET port input
	uint8 input_USB      = 4 #  USB data input
	uint8 input_CAN      = 5 #  CAN data input
